### PR TITLE
Added text utilities

### PR DESCRIPTION
**Pull request by nishasen**

### CODE REVIEW

**Comment by nishasen** [comment]
https://deploy-preview-17--nishasen-nembus-ui.netlify.app/html_pages/textutilities

File: HTML_Pages/textUtilities.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+                </code>
+            </pre>
+        </section>
+        <h3>Font weight</h3>
+        <section class="comp-display-box-ver">
+           <div class="p3 font-weight-black">This text has 900 font weight</p>
+           <div class="p3 font-weight-bold">This text has 700 font weight</p>
+           <div class="p3 font-weight-semibold">This text has 600 font weight</p>
+           <div class="p3 font-weight-regular">This text has 400 font weight</p>
+           <div class="p3 font-weight-light">This text has 300 font weight</p>
```
Comment (ttahm3d) [inline HTML_Pages/textUtilities.html:133]: Like the way you have given examples.

File: HTML_Pages/textUtilities.html [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                <code class="html">
+                   
+                </code>
+            </pre>
+        </section>
+        <h3>Centered text</h3>
+        <section class="comp-display-box-ver">
+           <h1 class="text-center">This is centered text</h1>
+           <h3 class="text-center">This is centered text</h3>
+           <p class="text-center">This is centered text</p>
```
Comment (ttahm3d) [inline HTML_Pages/textUtilities.html]: If it is same class, one example would be enough.

**Review by Sameony — APPROVED** [review]

**Review by ttahm3d — APPROVED** [review]
looks good. Imporvements can be done later on.

File: HTML_Pages/textUtilities.html (hunk 2 above)
Comment (nishasen) [reply HTML_Pages/textUtilities.html]: Okay will change this

**Pull request merged**